manageability-docker/TBH/LICENSE

```diff
@@ -0,0 +1,198 @@
+Press 'q' to quit once you have read the license.
```

Comment:
This looks like the proprietary license.  We need to use the Apache 2.0 license.  Did you find this one somewhere else in our code?  If so, we need to exchange it with the opensource one.  You can find the Apache 2.0 one at the root level.